src/components/login/Login.tsx

```diff
@@ -214,17 +209,23 @@ export const Login: React.FunctionComponent<LoginProps> = ({
                       </div>
                     )}
                     <div className="text-center">
-                      <Button
-                        color="primary"
-                        variant="contained"
-                        size="large"
-                        type="submit"
-                        disabled={!loginType}
-                        onSubmit={handleLogin}
-                        className="wide-button"
-                      >
-                        {t('common.logIn')}
-                      </Button>
+                      {isLoading ? (
```

Comment:
I see this is either displaying the loading Circular or the button. can we make it absolute position so that button is disable on loading and the Circular bar is on top of the page ?
- this is to keep consistency of displayed elements on the page.